resources/assets/components/utilities/SitewideBanner/SitewideBanner.js

```diff
@@ -4,6 +4,20 @@ import React, { useRef, useEffect } from 'react';
 import excludedPaths from './config';
 import SitewideBannerContent from './SitewideBannerContent';
 
+const checkExcludedPathname = pathname => {
+  for (let i = 0; i < excludedPaths.length; i += 1) {
+    if (excludedPaths[i] === pathname) {
+      return true;
+    }
+    if (excludedPaths[i].includes('*')) {
+      if (pathname.includes('/us/quiz-results/')) {
```

Comment:
Instead of hardcoding the quiz results path here, we should be checking for the substring before the `*` :
```suggestion
      if (pathname.includes(excludedPaths[i].slice(0, -1)) {
```

Reply:
This assuming that the wildcard is appearing at the end of the string, which I think is fine for our purposes -- we'll just need to spell it out in docs that we only support wildcards at the end (not something like `us/account/*/subscriptions`)

Reply:
Wouldn't this to the contrary of https://github.com/DoSomething/phoenix-next/pull/2188#discussion_r433518187 potentially include a base category (`/us/campaigns/`) when we want to exclude nested pages (`/us/campaigns/*`)?

Reply:
Hmm maybe we could use some form of regex to be extra sure in this case?
Something roughly like: 
```js
excludedPaths[i].slice(0, -1).match(`pathname${(\/)?(^[\?]).+}`)
```

Which would make sure the pathname isn't a base category 🤔

Reply:
> Wouldn't this to the flip side of #2188 (comment) potentially include a base category (/us/campaigns/) when we want to exclude nested pages (/us/campaigns/*)

Yep, good catch. regex sounds like itcould work...  i also wonder about checking the length of `pathname` in this conditional to check whether it's a nested page to avoid using regex (in interest of keeping this readable)

Reply:
That's an interesting idea! So basically ensuring that it's longer than the excluded pathname which means it _has_ a sub path?

Reply:
@aaronschachter so essentially `pathname.includes(excludedPaths[i].slice(0, -1) && pathname.length > excludedPaths[i].length`

Reply:
that looks like it should work!